packages/lib/passport-interface/src/CredentialManager.ts

```diff
@@ -103,13 +108,27 @@ export class CredentialManager implements CredentialManagerAPI {
     return undefined;
   }
 
+  private getCurrentCacheId(): string {
+    return (this.pcds.getPCDsByType(EmailPCDTypeName) as EmailPCD[])
+      .map((p) => p.claim.emailAddress)
+      .join(":");
+  }
+
```

Comment:
probably needs a comment - basically this makes the cache come up with nothing if the entry was generated when the user had a diff set of emails on their account than before, as derived from the set of email PCDs in their zupass.